Sources/MongoSwift/MongoClient.swift

```diff
@@ -278,6 +278,96 @@ public class MongoClient {
         return MongoDatabase(name: name, client: self, options: options)
     }
 
+    /**
+     * Starts a `ChangeStream` on a `MongoClient`. Allows the client to observe all changes in a cluster - excluding
+     * system collections i.g. "config", "local", and "admin" databases. By default, the type `CollectionType` is
+     * associated with the `fullDocument` field in the `ChangeStreamsDocument`.
+     * - Parameters:
+     *   - Pipeline: The pipeline of stages to append to an initial `ChangeStream` stage.
+     *   - Options: An optional `ChangeStreamOptions` to use on the initial `ChangeStream` stage.
+     *   - Session: An optional `ChangeStream` to use with this change stream.
+     * - Returns: a `ChangeStream` on all collections in all databases in a cluster.
+     * - Throws:
+     *   - `ServerError.commandError` if an error occurs on the server while creating the cursor.
+     *   - `UserError.invalidArgumentError` if the options passed formed an invalid combination or the pipeline passed
+     *     is invalid.
+     *   - `UserError.invalidArgumentError` if the `_id` field is projected out of the change stream documents by the
+     *     pipeline.
+     * - SeeAlso: https://docs.mongodb.com/manual/reference/system-collections/
```

Comment:
maybe worth also adding a seealso link to all of these docstrings that points to the page on change streams:
https://docs.mongodb.com/manual/changeStreams/
and maybe another linking to allowed aggregation stages for the pipeline:
https://docs.mongodb.com/manual/meta/aggregation-quick-reference/

(applies to all the methods)